src/VoerroTagsInput.vue

```diff
@@ -82,7 +82,7 @@
 
                 <li v-for="(tag, index) in searchResults"
                     :key="index"
-                    v-html="tag[textField]"
+                    v-html="tag[displayField] || tag[textField]"
```

Comment:
What if `tag[displayField]` is `null` or `0` or an empty string? Do you think it's appropriate to default to `tag[textField]` in these cases?

Reply:
I do think it's appropriate behavior as the `displayField` is meant to be an override of the `textField` for the dropdown list, but not for the tags in the input, so the `textField` data should always be passed anyways. If the `displayField` is `null`, `0` or empty then it just falls back to to the `value` / custom `textField` property that is already part of the package and functions normally.

I have been using this code successfully as is with and without the `displayField` tag passed through. By the way, that's just the field name I happened to come up with, but happy to change it to something else.

Reply:
I get what you're saying, but I believe a `0` and an empty string should not be considered as "no value", these are actually valid values. If you want to mark no value, pass a `null`. This is more of a general programming approach, not specific to this plugin. So, I would explicitly check `tag[displayField]` for `undefined` and `null`.

As for the naming, I can't think of a better name off the top of my head. This would only be used in the dropdown mode, so maybe it should be something more specific to that, or maybe not, I'm not sure. `displayField` is not bad or anything to my taste.